### DISCUSSION TITLE
Release 3.8.4

### DISCUSSION THREAD

**Discussion by Smolevich** [jenssegers/laravel-mongodb#2264] [General]
- Add doesntExist to passthru https://github.com/jenssegers/laravel-mongodb/pull/2194
- Fix getRelationQuery breaking changes https://github.com/jenssegers/laravel-mongodb/pull/2263
- Add Model query whereDate support https://github.com/jenssegers/laravel-mongodb/pull/2251
- Add transaction free deleteAndRelease() method https://github.com/jenssegers/laravel-mongodb/pull/2229
- Add setDatabase to Jenssegers\Mongodb\Connection https://github.com/jenssegers/laravel-mongodb/pull/2236
- Check dates against DateTimeInterface instead of DateTime https://github.com/jenssegers/laravel-mongodb/pull/2239
- Move from psr-0 to psr-4 https://github.com/jenssegers/laravel-mongodb/pull/2247
- Apply fixes produced by php-cs-fixer https://github.com/jenssegers/laravel-mongodb/pull/2250

<hr /><em>This discussion was created from the release <a href='https://github.com/jenssegers/laravel-mongodb/releases/tag/3.8.4'>Release 3.8.4</a>.</em>

**Comment by pd-0120** [comment]
I am currently running on laravel 6.x and laravel mongodb 3.6.
And I am facing issue with whereDate,
which can see that it is been released in 3.8 version.
Can you please also update in the 3.6 ?
If not possible to do so, hoe can I implement in my laravel mongo 3.6 ?
Thank you.

**Reply by divine** [reply]
Hello,

We will update it as soon as possible whenever will have time.

Thanks!
